tools/ndocs-project/Data/FileInfo.nd

```diff
@@ -1,521 +1,671 @@
-1.52
```

Comment:
Is this file supposed to be in here? It seems auto generated and environment dependent.

Reply:
As far as I can see FileInfo.nd is:
```
An index of the state of the files as of the last parse.  Used to determine if files were added, deleted, or changed.
```
https://www.naturaldocs.org/archive/documentation/1.4/files/Modules/NaturalDocs/Project-pm.html#FileInfo.nd

Because ndocs updates only changed files during its rebuild
```
Natural Docs' build process is differential, which means each run it looks for what changed since its last run and then updates only those parts of the documentation.  This means that later runs are much faster than the first so you can run it often as you work.
```
https://www.naturaldocs.org/getting_started/building_the_documentation/

Reply:
Should be in. gitignore then.

Reply:
I say keep it, it'll be useful once we do it via CI.

Reply:
It will be useful once we do it via CI yes, but with CI filepaths. These are environment dependent file paths. So currently that file is useless, we should re-add it with CI tho yes.

Reply:
I agree with Dedmen.

Reply:
Alright, I see what you mean.